### PR TITLE
[v6r19] fts and GSI from repo

### PR DESCRIPTION
**Pull request by fstagni**
(not deserving a release note!)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,8 @@
-GSI>=0.6.4
+# From repo
+git+https://:@gitlab.cern.ch:8443/fts/fts-rest.git#egg=fts3
```
Comment (fstagni) [inline requirements.txt]: No, I'd rather not. This file is only used for development and for running the unit tests. Indirectly, it'd end up being used for debugging new versions of FTS before making it to a release. So, I prefer leaving the "last" version.

Comment (chaen) [inline requirements.txt]: Ah good point !The others give a minimal version !

**Comment by chaen** [comment]
review OK

**Comment by fstagni** [comment]
I only added some test/documentation stuff

File: Core/Utilities/Platform.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 """
 Compile the externals
```
Comment (atsareg) [inline Core/Utilities/Platform.py]: This doc string is certainly a copy/paste from somewhere else ;)

**Pull request merged**